app/MMB/src/scenes/passenger/Passenger.js

```diff
@@ -3,30 +3,16 @@
 import * as React from 'react';
 import { graphql, createFragmentContainer } from '@kiwicom/mobile-relay';
 import { Translation, DateFormatter } from '@kiwicom/mobile-localization';
-import { SimpleCard, StyleSheet, Text, Color } from '@kiwicom/mobile-shared';
+import { SimpleCard, StyleSheet, Text } from '@kiwicom/mobile-shared';
 import idx from 'idx';
 import { View } from 'react-native';
 import startCase from 'lodash/startCase';
 
 import { SeparatorFullWidth } from '../../components/Separators';
 import VisaInformation from './visa/VisaInformation';
 import type { Passenger_passenger as PassengerType } from './__generated__/Passenger_passenger.graphql';
-
-const Row = ({ children }: {| children: React.Node |}) => (
-  <View style={styles.row}>{children}</View>
-);
-
-type ColumnProps = {|
-  title: React.Element<typeof Translation>,
-  value: React.Element<typeof Translation>,
-|};
-
-const Column = (props: ColumnProps) => (
-  <View style={styles.column}>
-    <Text style={styles.titleLight}>{props.title}</Text>
-    {props.value}
-  </View>
-);
+import Row from './Row';
+import Column from './Column';
```

Comment:
This is sooo common. Maybe we should extract it to our new package with our basic building blocks? 🙂

Reply:
I guess, but, right now it is just being used by passenger. We could probably make it more generic and move it to shared when needed.